examples/KDA/FLA_KDA/cumsum.py

```
@@ -0,0 +1,468 @@
+
+@input_guard
+def chunk_local_cumsum(
+    g: torch.Tensor,
+    chunk_size: int,
+    reverse: bool = False,
+    scale: float = None,
+    cu_seqlens: torch.Tensor  = None,
+    head_first: bool = False,
+    output_dtype: torch.dtype  = torch.float,
+    chunk_indices: torch.LongTensor  = None,
+    **kwargs,
```

Ruff (0.14.10)

Unused function argument: `kwargs`

(ARG001)